apps/src/applab/rgbcolor.js

```diff
@@ -189,6 +190,18 @@ module.exports = function(color_string)
             }
         },
         {
+          re: /^rgba\((\d{1,3}),\s*(\d{1,3}),\s*(\d{1,3}),\s*([0-9.]+)\)$/,
```

Comment:
Probably not an issue, but if we want to get super-specific, this accepts the string "rgba(0, 0, 0, .)"
I think this would be more specific?
```javascript
/^rgba\(\s*(\d{1,3})\s*,\s*(\d{1,3})\s*,\s*(\d{1,3})\s*,\s*
((?:\d+(?:\.\d+)?)|(?:\.\d+))
\s*\)$/